README.md

```diff
@@ -558,35 +547,31 @@ Public folder parameters
   - Disable favicon support by supplying `"none"` *[String]* to this parameter.
 - `symlinksToStatics`: Array of folders from `staticsRoot` to make symlinks to in your public folder, formatted as either `"linkName: linkTarget"` (whitespace optional) or simply `"linkName"` if the link target has the same name as the desired link name.
   - Default: *[Array]* of *[Strings]*
-
-
-```json
-[
-  "css: .build/css",
-  "images",
-  "js: .build/js"
-]
-```
+      ```json
+      [
+        "css: .build/css",
+        "images",
+        "js: .build/js"
+      ]
+      ```
 
 - `versionedStatics`: If set to true, Roosevelt will prepend your app's version number from `package.json` to your statics URLs. Versioning your statics is useful for resetting your users' browser cache when you release a new version.
   - Default: *[Boolean]* `false`.
 - `versionedCssFile`: If enabled, Roosevelt will create a CSS file which declares a CSS variable containing your app's version number from `package.json`. Enable this option by supplying an object with the member variables `fileName` and `varName`.
   - Default: `undefined`.
   - Example usage (with roosevelt-less): *[Object]*
-
-
-```json
-{
-  "fileName": "_version.less",
-  "varName": "appVersion"
-}
-```
-
-Assuming the default Roosevelt configuration otherwise, this will result in a file `statics/css/_version.less` with the following content:
-
-```less
-/* do not edit; generated automatically by Roosevelt */ @appVersion: '0.1.0';
-```
+      ```json
+      {
+        "fileName": "_version.less",
+        "varName": "appVersion"
+      }
+      ```
+
+  - Assuming the default Roosevelt configuration otherwise, this will result in a file `statics/css/_version.less` with the following content:
```

Comment:
I made this a bullet, not sure if that is something you wanted I can reverse if necessary